### PR TITLE
Fix #892: Highfi profile list

### LINKED ISSUE #892: Highfi ProfileListActivity
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/ddc3dadf-10e4-45f6-aeea-eb8a330e00ff/Settings-1-Profile-1/

https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0a5af83d-4b03-4a71-bf01-0c0a6e97e0c2/RG-MN-Register-Empty-34/

The activity should pass accessibility test.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #892 

## Mock:
https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/ddc3dadf-10e4-45f6-aeea-eb8a330e00ff/Settings-1-Profile-1/
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0a5af83d-4b03-4a71-bf01-0c0a6e97e0c2/RG-MN-Register-Empty-34/

## Screenshot:
Screenshot_1585313573
Screenshot_1585313575
Screenshot_1585313691
Screenshot_1585313698

## Known design changes:
- The shadow for the image view is shown to be `#4D000000` in mocks but actually that was not working correctly. Now this color is exactly same to `#B2B2B2` without transparency. So I used `#B2B2B2` and its other transparent versions for shadow.
- The accessibility test fails when we show `Administrator` string but that is fine because the `Administrator` string is meant to be a secondary string and we don't want learner's focus on that string.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/settings/profile/ProfileListActivityPresenter.kt [hunk 1]
```diff
@@ -4,6 +4,8 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.databinding.DataBindingUtil
 import org.oppia.app.R
 import org.oppia.app.activity.ActivityScope
+import org.oppia.app.completedstorylist.CompletedStoryListActivity
+import org.oppia.app.databinding.CompletedStoryListFragmentBinding
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/settings/profile/ProfileListActivityPresenter.kt]: Remove unused imports.

File: app/src/main/res/layout-land/profile_list_profile_view.xml [hunk 2]
```diff
@@ -14,51 +14,53 @@
...
       android:layout_height="wrap_content"
-      android:layout_marginStart="16dp"
-      android:orientation="vertical"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp"
+      android:ellipsize="end"
+      android:fontFamily="sans-serif"
+      android:maxLines="1"
+      android:text="@{profile.name}"
+      android:textColor="@color/black_87"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/profile_list_profile_view.xml]: Change this color to oppiaPrimaryTextDark.

File: app/src/main/res/layout/profile_list_profile_view.xml [hunk 3]
```diff
@@ -14,51 +14,53 @@
...
       android:layout_height="wrap_content"
-      android:layout_marginStart="16dp"
-      android:orientation="vertical"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp"
+      android:ellipsize="end"
+      android:fontFamily="sans-serif"
+      android:maxLines="1"
+      android:text="@{profile.name}"
+      android:textColor="@color/black_87"
```
Comment (veena14cs) [inline app/src/main/res/layout/profile_list_profile_view.xml]: Ditto.

**Review by veena14cs — APPROVED** [review]
LGTM beside nit change.

File: app/src/main/java/org/oppia/app/settings/profile/ProfileListActivityPresenter.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/settings/profile/ProfileListActivityPresenter.kt]: Removed.

File: app/src/main/res/layout-land/profile_list_profile_view.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout-land/profile_list_profile_view.xml]: Done.

File: app/src/main/res/layout/profile_list_profile_view.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/profile_list_profile_view.xml]: Done.

**Pull request merged**